### DISCUSSION TITLE
Cellebrite UFDR

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#351] [Q&A]
Machine translated:
Good afternoon Nassif,
I processed a telematic break from Apple on Cellebrite, generated the UFDR, and then the report in HTML.
I spent both as a parameter on iped in different indexes to compare. It is important to note that I went to the CategoriesToExpand.txt file in the Default profile and uncommented everything, since my intention is to copy the Exported folder to a Big Data tool and analyze the data together with others from the same investigation.
![Comparar-Exported-UFDR-HTML](https://user-images.githubusercontent.com/55517932/106013353-20b71980-609b-11eb-8d1d-12de6096690e.png)
The number of files exported the most when it comes to the HTML report is noteworthy, is this behavior normal?

Another point, neither of the two processes was legal to categorize. Do you have any suggestions on how to deal with this categorization issue?
![iped-categoria](https://user-images.githubusercontent.com/55517932/106013889-adfa6e00-609b-11eb-84e7-3cd13fc5903d.png)

Thanks.

**Comment by lfcnassif** [comment]
Please post in english.

Just UFDR and XML UFED reports are supported, and you must pass the *.ufdr file as argument to IPED -d option. If you pass the ufdr parent folder (or an unsupported Html report) phone specific categories will not be recognized.

I think you meant CategoriesToExport.txt instead of CategoriesToExpand.txt. Even if you edited the right file, just actual files are exported (pdf, doc, jpg, etc) if I remember, database records such as calls and SMS messages are not exported and are stored just into the case index, they are not files actually.

**Reply by thiagocaparelli** [reply]
First of all, thanks for the attention.

I have passed the UFDR file as argument like you suggested. And yet the categorization didn't look good.

**Reply by lfcnassif** [reply]
hum... possibly because phone categories are missing in CategoriesToExport.txt file, so when entries in that file were uncommented, just desktop specific categories were added to the case. Try to comment out all categories again, so all categories will be added to the case. Or try to copy phone specific categories from CategoriesByTypeConfig.txt to CategoriesToExport.txt.

**Reply by lfcnassif** [reply]
Just opened #353

**Reply by thiagocaparelli** [reply]
Ok. I'll try it out. Thanks.

**Reply by thiagocaparelli** [reply]
It worked! Thanks a lot.
